eslint.config.mjs

```diff
@@ -51,6 +51,9 @@ export default tseslint.config(
         languageOptions: {
             parserOptions: {
                 warnOnUnsupportedTypeScriptVersion: false,
+                projectService: true,
+                defaultProject: "./scripts/tsconfig.json",

```

Comment:
This file does not cover our misc config files at the root (maybe it should?), but using it here is okay given it's a config with reasonable settings / noEmit.